gaphor/services/copyservice.py

```diff
@@ -131,9 +132,11 @@ def copy_action(self):
             for i in items:
                 copy_items.append(i)
             self.copy(copy_items)
-            view.get_action_group("win").lookup_action("edit-paste").set_enabled(
-                bool(self.copy_buffer)
-            )
+            win_action_group = view.get_action_group("win")
+            if win_action_group is not None:
```

Comment:
Funny you’re seeing this issue and not me 🤔. I think the ‘is not None’ part is not needed. It’ll do a none check anyway.